src/components/user/SharedAuth.tsx

```diff
@@ -28,13 +24,11 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-function checkAuthRedirect(authRedirect: string) {
+function checkAuthRedirect(authRedirect: string): boolean {
   try {
-    const url = new URL(authRedirect);
+    new URL(authRedirect);
 
-    return AUTH_ALLOWED_HOSTNAMES.some(
-      allowedHostname => allowedHostname === url.hostname
-    );
+    return true;
```

Comment:
Does this mean, we are bypassing auth for now?

Reply:
No, it just doesn't really makes sense to check what is the callback URL as we don't share any information while doing the redirect, the token is set in the browser so after redirect it should be already available. The next part may be to make sure the cookie is not included when doing request to 3rd party domains.